style.css

```diff
@@ -88,3 +87,285 @@ p {
+}
+
+.stories {
+  font-family: 'Inter', sans-serif;
+  font-size: 24px;
+  font-weight: 600;
+  text-align: center;
+  color: #3a4a42;
+}
+
+.project-skills-wrapper {
+  display: flex;
+  list-style: none;
+  align-items: center;
+  gap: 8px;
+}
+
+.project-skill {
+  font-family: 'Inter', sans-serif;
+  font-size: 12px;
+  font-weight: 400;
+  padding: 8px 12px;
+  background-color: #ebf0ee;
+  color: #3a4a42;
+  border-radius: 4px;
+}
+
+.see-project-button {
+  font-family: 'Inter', sans-serif;
+  color: white;
+  background-color: #36b37f;
+  font-size: 17px;
+  font-weight: 500;
+  text-align: center;
+  border: none;
+  border-radius: 4px;
+  padding: 10px;
+}
+
+.see-project-button:active {
+  cursor: pointer;
+  background-color: #008552;
+}
+
+.see-project-button:hover {
+  cursor: pointer;
+  background-color: #36b37f;
+  box-shadow: rgba(54, 179, 127, 0.24);
+}
+
+.see-project-button:visited {
+  background-color: #008552;
+  font-weight: 500;
+}
+
+.about-me {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 1.6rem;
+  padding: 4rem 1rem;
+  background: url('images/Mask-Group.png'), url('images/illustration-about-me-mobile.png');
+  background-repeat: no-repeat no-repeat;
+  background-position: right top, left bottom 3.25rem;
+}
+
+.headding {
+  text-align: center;
+  font-family: 'Inter', sans-serif;
+  color: #172b4d;
+  font-weight: 700;
+  font-size: 40px;
+}
+
+.about-para {
+  color: #42526e;
+  font-weight: 400;
+  font-size: 20px;
+  text-align: center;
+}
+
+.resume-btn {
+  font-family: 'Inter', sans-serif;
+  font-weight: 500;
+  display: flex;
+  background-color: #36b37f;
+  font-size: 17px;
+  width: max-content;
+  color: white;
+  padding: 12px;
+  border-radius: 4px;
+  align-self: center;
+}
+
+.wrap-languages {
+  background-color: #ebf0ee;
+  border: 1px solid #c1c7d0;
+  background-image: url(images/icon-languages.png);
+  background-repeat: no-repeat;
+  background-position: center top 2rem;
+}
+
+.cards-languages {
+  display: flex;
+  list-style: none;
+  justify-content: center;
+  gap: 8px;
+}
+
+.languages {
+  display: flex;
+  font-family: 'Inter', sans-serif;
+  font-weight: 600;
+  font-size: 24px;
+  color: #344563;
+  justify-content: center;
+  padding-bottom: 24px;
+  padding-top: 8.5rem;
+}
+
+.final-cards-languages {
+  font-family: 'Inter', sans-serif;
+  font-size: 12px;
+  font-weight: 600;
+  padding: 12px;
+  background-color: white;
+  color: #36b37f;
+  border-radius: 4px;
+  margin-bottom: 4rem;
+}
+
+.wrap-frameworks {
+  background-color: #ebf0ee;
+  border: 1px solid #c1c7d0;
+  background-image: url(images/icon-frameworks.png);
+  background-repeat: no-repeat;
+  background-position: center top 2rem;
+  margin-top: 1.5rem;
+}
+
+.frameworks {
+  display: flex;
+  font-family: 'Inter', sans-serif;
+  font-weight: 600;
+  font-size: 24px;
+  color: #344563;
+  justify-content: center;
+  padding-bottom: 24px;
+  padding-top: 8.5rem;
+}
+
```

Comment:
- [ ] Please add some margin from left and right side to your tech stack cards to match with the provided design


|Your design|Figam design|
|----|-----|
|<img width="280" alt="Screenshot 2023-06-10 at 8 47 41 AM" src="https://github.com/MoliBravo/Portfolio_setup_and_mobile_first/assets/44798044/467e712e-87c2-40af-a371-d061bf7c8189">|<img width="158" alt="Screenshot 2023-06-10 at 8 47 19 AM" src="https://github.com/MoliBravo/Portfolio_setup_and_mobile_first/assets/44798044/06372e9b-580f-463f-9144-dbf74578b0da">|